java/memory/memory-unsafe/src/test/java/org/apache/arrow/memory/unsafe/TestUnsafeAllocationManager.java

```diff
@@ -15,20 +15,25 @@
  * limitations under the License.
  */
 
-package org.apache.arrow.memory;
+package org.apache.arrow.memory.unsafe;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
+import org.apache.arrow.memory.AllocationManager;
+import org.apache.arrow.memory.ArrowBuf;
+import org.apache.arrow.memory.BufferLedger;
+import org.apache.arrow.memory.RootAllocator;
+import org.apache.arrow.memory.unsafe.UnsafeAllocationManager;
 import org.junit.Test;
 
 /**
  * Test cases for {@link UnsafeAllocationManager}.
  */
 public class TestUnsafeAllocationManager {
 
-  private BaseAllocator createUnsafeAllocator() {
-    return new RootAllocator(BaseAllocator.configBuilder().allocationManagerFactory(UnsafeAllocationManager.FACTORY)
+  private RootAllocator createUnsafeAllocator() {
```

Comment:
Ideally we never refer to RootAllocator (or BaseAllocator) as a type, only BufferAllocator

Reply:
Changed to return BufferAllocator instead.